### DISCUSSION TITLE
[Feature]  Add photos to offline cache manually

### DISCUSSION THREAD

**Discussion by ghost** [immich-app/immich#5993] [Feature Request]
### The feature

After wiping or switching to another smartphone, all my photos are on the server. None on the smartphone. I often have no or poor internet, so I want to make some albums or some/all photos available for offline cache.
Please add the option to handle the offline cache. Thanks

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7627
